config/routes.rb

```diff
@@ -4,9 +4,11 @@
   match '/logout', to: 'sessions#logout', via: [:delete, :get]
   if Figaro.env.enable_sso_integration == "true"
     devise_for :users, controllers: { omniauth_callbacks: 'omniauth_callbacks'}
-  end
-  as :user do
-    get 'signin', to: 'devise/sessions#new', as: :new_user_session
+    as :user do
+      get 'signin', to: 'devise/sessions#new_sso', as: :new_user_session
```

Comment:
is this will impact existing sso login ways?

Reply:
I think no mas, this is intended to fix development environment, iirc this causes dev env to not to be able to run since it still always redirect back to SSO

Reply:
noted